cache/class-vary-cache.php

```diff
@@ -86,6 +90,11 @@ public static function set_group_for_user( $group, $value ) {
 		// TODO: only send header if we added or changed things
 		// TODO: don't set the cookie if was already set on the request
 		// validate, process $group, etc.
+		if ( strpos( $group, self::GROUP_SEPARATOR ) !== false || strpos( $group, self::VALUE_SEPARATOR ) !== false ||
+			strpos( $value, self::GROUP_SEPARATOR ) !== false || strpos( $value, self::VALUE_SEPARATOR ) !== false ) {
+				trigger_error( 'Cannot use the group or value separator text in the group/value name', E_USER_WARNING );
```

Comment:
Same as earlier, the error message should be a bit more explicit about what happened and why:

> Failed to add user to group; cannot use the group delimiter value (`%s`) in the group name...